packages/react-components/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -74,135 +51,98 @@ export default {
+  </Tooltip>
 );
-
 Default.args = {
-  placement: 'bottom',
   isVisible: true,
-  theme: 'default',
-  triggerOnClick: false,
-  arrowOffsetY: 0,
-  arrowOffsetX: 0,
-  offsetMainAxis: 8,
-  withFadeAnimation: true,
-  transitionDuration: 200,
-  transitionDelay: 0,
-  hoverOutDelayTimeout: 100,
 };
+Default.decorators = [
+  (Story: StoryFn) => (
+    <div className="tooltip-story default">
+      <Story />
+    </div>
+  ),
+];
 
-export const TooltipInfo = (): React.ReactElement => (
-  <div className="tooltip-preview-container">
-    <Tooltip
-      isVisible
-      triggerRenderer={() => (
-        <div>
-          <Button
-            icon={<Icon source={ChevronDown}></Icon>}
-            iconPosition={'right'}
-          >
-            Open Tooltip
-          </Button>
-        </div>
-      )}
-    >
-      <Info
-        header="Header - concise and clear"
-        text="Tooltip content is used to explain the details of elements or features."
-        closeWithX
-      />
-    </Tooltip>
-  </div>
+export const TooltipInfo = (args: ITooltipProps): React.ReactElement => (
+  <Tooltip {...args} triggerRenderer={<Button>Trigger</Button>}>
+    <Info
+      header="Header - concise and clear"
+      text="Tooltip content is used to explain the details of elements or features."
+      closeWithX
+      handleCloseAction={noop}
+    />
+  </Tooltip>
 );
+TooltipInfo.args = {
+  isVisible: true,
+};
+TooltipInfo.decorators = [
+  (Story: StoryFn) => (
+    <div className="tooltip-story info">
+      <Story />
+    </div>
+  ),
+];
 
-export const TooltipInteractive = (): React.ReactElement => (
-  <div className="tooltip-preview-container">
-    <Tooltip
-      isVisible
-      className="tooltip-interactive"
-      triggerRenderer={() => (
-        <div>
-          <Button
-            icon={<Icon source={ChevronDown}></Icon>}
-            iconPosition={'right'}
-          >
-            Open Tooltip
-          </Button>
-        </div>
-      )}
-    >
-      <Interactive
-        header="Header - concise and clear"
-        image={{
-          src: beautifulImage,
-          alt: 'image',
-        }}
-        text="Tooltip content is used to explain the details of elements or features."
-        closeWithX
-        primaryButton={{
-          handleClick: noop,
-          label: 'Primary Button',
-        }}
-        secondaryButton={{
-          handleClick: noop,
-          label: 'Secondary',
-        }}
-      />
-    </Tooltip>
-  </div>
+export const TooltipInteractive = (args: ITooltipProps): React.ReactElement => (
+  <Tooltip {...args} triggerRenderer={<Button>Trigger</Button>}>
+    <Interactive
+      header="Header - concise and clear"
+      image={{
+        src: beautifulImage,
+        alt: 'image',
+      }}
+      text="Tooltip content is used to explain the details of elements or features."
+      closeWithX
+      primaryButton={{
+        handleClick: noop,
+        label: 'Primary Button',
+      }}
```

Comment:
I think that we normally should allow to pass any button component here and in other similar situations, but we can leave it for version 2.0 cause it will be beaking change. 🤔